From 242536e1acbaa756e261c229668d09f405316ce2 Mon Sep 17 00:00:00 2001
From: Kevin Figueroa <kfdevart@disroot.org>
Date: Sat, 29 Mar 2025 18:05:20 -0400
Subject: [PATCH] Remove unnecessary steps for XBPS packaging

clean_theme is removed because xbps-src already takes care of this and it's an obstacle for installing multiple variants (i.e. black and nord) to DESTDIR that will later be separated in subpackages via vmove.

install_package is not required since sassc it's already in makedepends.
---
 install.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install.sh b/install.sh
index e3b0006..9d17651 100755
--- a/install.sh
+++ b/install.sh
@@ -836,7 +836,7 @@ else
       echo -e "\nNeed root permission !"
     fi
   else
-    clean_theme && install_package && sass_temp && gnome_shell_version && install_theme
+    sass_temp && gnome_shell_version && install_theme
 
     if [[ "$libadwaita" == 'true' ]]; then
       uninstall_link && link_theme
-- 
2.49.0

